oppia/oppia-android
# Fix part of #1460: Exploration file names updated

## Explanation


This PR fixes part of #1460.

## Checklist
- [x] Introduce `topics.json` file so that we can dynamically load explorations.
- [x] Renames all exploration files based on `exploration_id` so that the code be made simpler.
- [x] Structure of exploration json files needs to be updated.

## How to test:
Run the app and check if explorations are working or not.
To test if all explorations are still visible correctly working or not, in `HomeFragmentPresenter`, add this line in `handleCreateView`. Note, you will need to inject `StoryProgressTestHelper` in constructor.

```
 logHomeActivityEvent()
 .
 .
 storyProgressTestHelper.markFullTopicProgressForFractions(profileId, false)
 storyProgressTestHelper.markTwoPartialStoryProgressForRatios(profileId, false)
 .
 .
 welcomeViewModel = WelcomeViewModel()
```

## Pending (will be done in future PRs)
- [ ] Skills json file needs to be generalised. Currently we are using `fractions_skills.json` and we need to shift this to id based naming.
- [ ] Questions json files needs to be generalised.
- [ ] Structure of question json files needs to be updated.

## Checklist
